logdetective/server/server.py

```diff
@@ -48,6 +47,18 @@
 FAILURE_LOG_REGEX = re.compile(r"(\w*\.log)")
 
 
+if SERVER_CONFIG.log.path:
+    logging.basicConfig(
```

Comment:
Using `logging.basicConfig()` means that we're setting the log configuration for ANYTHING that uses the python logging facility. That includes things like `requests` and `urllib3` at the low level. This can get a little noisy.

Can we restrict the logging setup to just the logger we're going to use?

Reply:
Good point, thank you.